cmd/PolicyGenerator/VERSION.txt

```diff
@@ -0,0 +1 @@
+v1.12.2
```

Comment:
What is the advantage of a separate file as opposed to defining the value in a constant variable in `main.go`?

Reply:
Mostly copy/paste, but it is easier to parse externally (like in the GitHub Action check). I'm not attached to it if you think a constant would be the way to go!

Reply:
I don't have a strong preference but one option is that we could define a constant like `const version = "VERSION"` and then in the GitHub release workflow, there could be a sed command run to replace `"VERSION"` with the GitHub tag.

That way we wouldn't have to update the version manually.

Reply:
I definitely like not having to update the version manually! And we could use similar logic in the subscription image build to inject a version.